components/DropdownLight/DropdownLight.jsx

```diff
@@ -236,56 +278,80 @@ const DropdownLight = ({ disabled, items, theme, placeholder, name }) => {
-        />
-      </Button>
-
-      {isOpen && (
-        <SelectionList theme={theme} ref={listOptions}>
-          {items.map((item, index) => (
-            <SelectionListItem
-              role="option"
-              theme={theme}
-              key={item?.value || item}
-              onClick={() => handleOnClickListItem(item)}
-              aria-posinset={index}
-              aria-selected={index === cursor}
-              tabIndex="-1"
-            >
-              {item?.label || item}
-
-              {(selectedItem === item?.value || selectedItem === item) && (
-                <CheckIcon theme={theme} />
+    <>
+      <ComponentWrapper theme={theme} skin={skin}>
+        <InputWrapper ref={wrapperRef}>
+          {label && (
+            <InputLabel error={error}>
+              {label}
+              {required && (
+                <RequiredMark skin={skin} aria-label="campo obrigatório">
+                  *
+                </RequiredMark>
               )}
-            </SelectionListItem>
-          ))}
-        </SelectionList>
-      )}
-    </Wrapper>
+            </InputLabel>
+          )}
+          <InputText
+            type="text"
+            hidden
+            skin={skin}
+            name={name}
+            defaultValue={selectedItem}
+            aria-label="selecione uma opção"
+            required={required}
+          />
+          <Button
+            aria-haspopup="true"
+            aria-label={
+              isOpen ? 'fechar lista de itens' : 'abrir lista de itens'
+            }
+            onClick={handleToggleDropdown}
+            theme={theme}
+            skin={skin}
+            error={error}
+            disabled={disabled}
+            ref={buttonRef}
+            id={id}
+          >
+            {itemLabel}
+            <ArrowIcon name={isOpen ? 'arrow_drop_up' : 'arrow_drop_down'} />
+          </Button>
+        </InputWrapper>
+
+        {isOpen && (
+          <SelectionList id={id} theme={theme} ref={listOptions}>
+            {items.map((item, index) => (
+              <SelectionListItem
+                role="option"
+                theme={theme}
+                key={item?.value || item}
+                onClick={() => handleOnClickListItem(item)}
+                aria-posinset={index}
+                aria-selected={index === cursor}
+                tabIndex="-1"
+              >
+                {item?.label || item}
+
+                {(selectedItem === item?.value || selectedItem === item) && (
+                  <CheckIcon theme={theme} />
+                )}
+              </SelectionListItem>
+            ))}
+          </SelectionList>
+        )}
+        {helperText && <HelperText>{helperText}</HelperText>}
+        {error && (
+          <InputErrorMessage theme={theme} skin={skin}>
+            {error}
+          </InputErrorMessage>
+        )}
+      </ComponentWrapper>
+    </>
   );
 };
 
 DropdownLight.propTypes = {
+  id: PropTypes.string,
```

Comment:
Where the explanations for each prop?

![image](https://user-images.githubusercontent.com/3389749/220738696-deeefd59-1637-489f-a2af-1ef82b1a2cbc.png)

Original Dropdown

![image](https://user-images.githubusercontent.com/3389749/220738849-2dab8dc5-77cf-4c2b-adb2-423f801580b3.png)

Reply:
There are things that I waited for all parts of the components to be ready and coupled, to then be adjusted

Reply:
The new props are this task. Then the explanation props and docs, should be here too.